version-compatibility/forkless-upgrade/README.md

```diff
@@ -0,0 +1,15 @@
+# Forkless upgrade tests
+
+This crate tests that state transition functions for all releases of `fuel-core` are backward compatible. 
+
+In addition, we also test that releases are forward-compatible unless we introduce a breaking change in the API.
+
+## Adding new test
+
+We need to add a new backward compatibility test for each new release. To add tests, we need to duplicate tests that are using the latest `fuel-core` and replace usage of the latest crate with a new release.
```

Comment:
Is there anything enforcing that we do this?

Reply:
Nope, and I don't know who to enforce it=)